project-base/storefront/components/Forms/Button/LinkButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { getButtonClassName } from './Button';
+import { ExtendedNextLink, ExtendedNextLinkProps } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
+import { twMergeCustom } from 'utils/twMerge';
+
+export type LinkButtonProps = {
+    isDisabled?: boolean;
+    isWithDisabledLook?: boolean;
+    size?: 'small' | 'medium';
+    variant?: 'primary' | 'secondary';
+} & ExtendedNextLinkProps;
```

Comment:
Can't you reuse these 4 props from the button component?

Reply:
Sure, good point, reused